src/fragments/lib-v1/storage/native_common/getting-started/common.mdx

```diff
@@ -134,4 +130,4 @@ Congratulations! You've uploaded a file to an s3 bucket. Check out the following
 
 <InlineFilter filters={['android', 'swift']}>
   - [Escape Hatch](/gen1/[platform]/prev/build-a-backend/storage/sdk/)
-</InlineFilter>
+</InlineFilter>
```

Comment:
Not sure what this diff is showing. Can you make sure this is rendering fine for iOS and Android?

Reply:
Yep, it's rendering in iOS and Android. The list of options looks a little funky because there's a space after `Using Lambda Triggers` but it looks like that space was there before.